src/policy/fees.h

```diff
@@ -175,12 +175,18 @@ static const double DEFAULT_DECAY = .998;
 /** Require greater than 95% of X feerate transactions to be confirmed within Y blocks for X to be big enough */
 static const double MIN_SUCCESS_PCT = .95;
 
+// Dogecoin: Smooth estimated fee so if it's close to 1.14.5 default, we use that default instead.
+// This avoids floating point math causing the estimate to drop just below the default inclusion
+// rate, while retaining the ability to use fees below 1.14.5 default.
+static const double SMOOTHING_TARGET = COIN / 100;
+static const double SMOOTHING_TOLERANCE = COIN / 100000;
+
```

Comment:
I think this artificially creates another developer enforced fee, except now we call it "smart". Per the logic in `CBlockPolicyEstimator::estimateSmartFee`, let's do the following hypothetical scenario:

1. The majority of the miners decide to run with `-blockmintxfee=0.001`, and 
2. There is a significant part of the relay network running with `-minrelaytxfee=0.001`, and
3. Resulting in the `mean` (effective txfee across a series of analyzed blocks) being `0.005`:

```
mean = 0.005
target = 0.01
tolerance = 0.00001
delta = mean - target = 0.005 - 0.01 = -0.005

is_intolerable = delta < tolerance = -0.005 < 0.00001 = true

smartfee = is_intolerable ? target : mean = true ? 0.01 : 0.005 = 0.01
```

Hence, you will cap the minimum fee again with your hardcoded value. Perhaps you can replace the hardcoded SMOOTHING_TARGET with the value given to `-minrelaytxfee` and make the tolerance a fraction of that?

Reply:
... please disregard the above, I missed the `abs()`, so that won't happen.

It still aligns with a hardcoded value though, so that part I'm still not sure about... because I think it makes it more confusing. Also note that `SMOOTHING_TOLERANCE` has the same value as `DEFAULT_INCREMENTAL_RELAY_FEE` from `policy.h`

Reply:
I've switched it to use the same block inclusion minimum the node has (including reading it from the CLI options), and then the error-tolerance value is 1/1000th of that. Hopefully that's better.

Reply:
That's better because that makes it configurable (and for miners it doesn't matter much because they anyway wouldn't pay fees to themselves, or pay fees to themselves - either way)